MagazineLayout/Public/MagazineLayout.swift

```diff
@@ -1144,9 +1137,10 @@ public final class MagazineLayout: UICollectionViewLayout {
     }
 
     guard
-      indexPath.section < modelState.numberOfSections(.beforeUpdates),
-      indexPath.item < modelState.numberOfItems(inSectionAtIndex: indexPath.section, .beforeUpdates) else
-    {
+      indexPath.section < modelStateBeforeBatchUpdates.numberOfSections,
+      indexPath.item < modelStateBeforeBatchUpdates.numberOfItems(
+        inSectionAtIndex: indexPath.section)
+    else {
       // On iOS 9, `layoutAttributesForItem(at:)` can be invoked for an index path of a new item
       // before the layout is notified of this new item (through either `prepare` or
```

Comment:
should we go ahead and clean up these comments referencing unsupported versions?